README.md

```diff
@@ -28,14 +129,22 @@ db.put('name', 'encoding-down', function (err) {
 
 ### `const db = require('encoding-down')(db[, options])`
 
-* `db` `abstract-leveldown` compatible db such as `leveldown`, `memdown`, `level-js` etc
-* `options.keyEncoding` (string) defaults to `'utf8'`
-* `options.valueEncoding` (string) defaults to `'utf8'`
+* `db` must be an [`abstract-leveldown`] compliant store
+* `options.keyEncoding` (string or object) defaults to `'utf8'`
+* `options.valueEncoding` (string or object) defaults to `'utf8'`
+
+`options` are passed to [`level-codec`](https://github.com/level/codec).
+
+## Custom encoding
 
-`options` are passed to [`level-codec`](https://github.com/level/codec), see [supported encodings](https://github.com/Level/codec#encodings) for more information.
+*todo*.
```

Comment:
Document the format in `level-codec` (most importantly: `encoding.buffer`), then link to it from here.